Sources/Fuzzilli/Fuzzer.swift

```diff
@@ -438,6 +461,12 @@ public class Fuzzer {
         return corpus.supportsFastStateSynchronization
     }
 
+    private func containsPoison(execution: Execution, differentialPoison: [String]) -> Bool {
```

Comment:
What does this do? Does it somehow mark samples as invalid so they aren't compared for differential results?

Reply:
Correct; some sample trigger behavior within the engines which renders any differentials invalid. E.g., the amount of stack memory differs between interpreter/jit, so one of them might overflow whereas the other doesn't. We must not compare the result of these executions as this would be a false positive.

Reply:
Makes sense, thanks! Maybe we should rename it to something more descriptive, e.g. `shouldExecutionBeIgnoredForDifferentialTesting`, and then I guess it can take the `differentialPoison` directly from the Configuration?